src/server/internalServer.cpp

```diff
@@ -913,7 +913,11 @@ std::unique_ptr<Response> InternalServer::handle_raw(const RequestContext& reque
     return Response::build_404(*this, request.get_full_url(), bookName, "", error_details);
   }
 
-  auto itemPath = request.get_url().substr(bookName.size()+kind.size()+6);
+  // Remove the beggining of the path:
+  // /raw/<bookName>/<kind>/foo
+  // ^^^^^          ^      ^
+  //   5      +     1  +   1   = 7
+  auto itemPath = request.get_url().substr(bookName.size()+kind.size()+7);
   if (itemPath[0] == '/') {
     itemPath = itemPath.substr(1);
   }
```

Comment:
Is this still needed? If yes, then one can also argue in favour of replacing the `if` with a `while`.

Reply:
No, I've removed it.
The itemPath must be exact (and nothing prevent a zim file to have a item starting with a `/`)